lib/extensions/date.rb

```diff
@@ -0,0 +1,8 @@
+# Blindly calculates end of workday using BusinessTime settings.
+# Does not consider weekends or holidays.
+class Date
+  def end_of_workday
```

Comment:
I think we should remove the monkey-patching before merging. Thoughts @harrisj, @baccigalupi?